### PR TITLE
Add status store

### PR DESCRIPTION
**Pull request by snakazawa**
I implemented player status store.
**The destination of store is `store/player_status.json`.** This implementation is very very simple.
Why?
I think that the better implementation is using DBMS, but it complicates this program by DB config, migration, scheme and so on.
Jukebox should keep simple.

### CODE REVIEW

**Comment by snakazawa** [comment]
fix #33

File: .gitignore [hunk 1]
```diff
@@ -55,4 +55,6 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
\ No newline at end of file
+.env
+
+/store/*.json
```
Comment (unblee) [inline .gitignore]: Please change
```
/store/*.json
```
to
```
/store/*
!/store/.gitkeep
```
I think this description is more suitable.

Comment (snakazawa) [inline .gitignore]: ok, I applied your comment :)

**Comment by unblee** [comment]
Umm...
I sent two URLs, but only one is added to the playlist.

**Comment by snakazawa** [comment]
a~ ok, you tried with `npm dev` ?

**Comment by snakazawa** [comment]
I solved the problem with merging master to this branch.
The cause of the problem is restarting server by nodeman when `store/player_status.json` is updated to save status (that file is watched by nodeman!!!).

**Comment by unblee** [comment]
Oh...
OK, I'll check it.

**Comment by unblee** [comment]
OK! 💯 
This time I confirmed it works in my environment.
Thanks!! 👍 👍

**Comment by unblee** [comment]
And thank you for your consideration. 🙏 
> Jukebox should keep simple.

**Pull request merged**